errors.go

```diff
@@ -26,6 +26,33 @@ var (
 	// ErrInvalidName is returned when the name option has invalid value.
 	ErrInvalidName = errors.New("invalid value for name")
 
+	// ErrMissingDirectory is returned when the path does not have an existing directory.
+	ErrMissingDirectory = errors.New("path needs to be an existing directory")
```

Comment:
I think that'll be hard because the naming isn't necessarily consistent (ie `ErrInvalidName` vs `ErrMissingDirectory`). And besides, this file will get pretty large once all the errors are in, so one will just ctrl-f via editor.

But I do think making sections for generic errors (ie `resource not found`) vs resource specific errors is useful! Will update